prov/gni/src/gnix_msg.c

```diff
@@ -314,27 +411,28 @@ static void __gnix_msg_copy_unaligned_get_data(struct gnix_fab_req *req)
 	int head_off, head_len, tail_len;
 	void *addr;
 
-	head_off = req->msg.send_addr & GNI_READ_ALIGN_MASK;
+	head_off = req->msg.send_info[0].send_addr & GNI_READ_ALIGN_MASK;
```

Comment:
So vectors including unaligned buffers are still not supported?  Should we make an issue for that?